flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/rules/datastream/DataStreamCalcRule.scala

```diff
@@ -46,6 +52,20 @@ class DataStreamCalcRule
       calc.getProgram,
       description)
   }
+
+  override def matches(call: RelOptRuleCall): Boolean = {
```

Comment:
This can be removed because the `RexOver` was removed from `Project` (and Calc`) during normalization.